### ISSUE TITLE
[Hub Menu] Menu items cannot be opened in VoiceOver

### ISSUE THREAD

**Issue by joshheald** [woocommerce/woocommerce-ios#13677]
## Description
Since WCiOS 17.9, it's not been possible to open items in the hub menu using VoiceOver (and potentially other accessibility technologies.)

This is due to the [transparent overlay modifier added to each menu item, adding a transparent NavigationItem](https://github.com/woocommerce/woocommerce-ios/blob/7813662efcc2e563dea4111b6d99172757f94ee1/WooCommerce/Classes/ViewRelated/Hub%20Menu/HubMenu.swift#L100), which is a workaround to prevent a second chevron being added to the menu items. That allows us to have different chevron behaviour, which can't otherwise be overridden in a list.

Unfortunately, the overlay also prevents the double-tap activate gesture in VoiceOver from working.

## Repro
1. Open the app
2. Navigate to the menu
3. Enable VoiceOver
4. Swipe right to get to a menu item (not the store switcher)
5. Double tap to activate
6. Observe that it simply reads the name again.

Expectation: the relevant menu item should open.

## Screenshots
https://github.com/user-attachments/assets/e552459f-9b88-49a1-83cc-03977dacd613

**Comment by dangermattic** [comment]
Thanks for reporting! 👍
